src/app/course/course-list/course-list.component.ts

```diff
@@ -1,15 +1,29 @@
-import { Component, OnInit } from '@angular/core';
-
-@Component({
-  selector: 'app-course-list',
-  templateUrl: './course-list.component.html',
-  styleUrls: ['./course-list.component.scss']
-})
-export class CourseListComponent implements OnInit {
-
-  constructor() { }
-
-  ngOnInit(): void {
-  }
-
-}
+import { Component, OnInit } from '@angular/core';
+import { COURSES } from '../mock-data';
+
+@Component({
+  selector: 'app-course-list',
+  templateUrl: './course-list.component.html',
+  styleUrls: ['./course-list.component.scss']
+})
+export class CourseListComponent implements OnInit {
+  courseList = COURSES;
+
+  ngOnInit(): void {
+    console.log('OnInit')
+  }
+
+
+  public deleteCourse(item): void {

```

Comment:
There is inconsistency with `public` modifiers in this class.
Class methods are public by default. You may skip `public` everywhere OR to write it everywhere, including `ngOnInit`. It's a matter of a project convention. You have to choose what way you prefer and stick to it for all the classes.

If you want to mark public methods explicitly, `public` should be applied to `ngOnInit`. If you prefer skipping default modifiers, remove `public` from all the methods.